playground.sh

```diff
@@ -64,140 +127,277 @@ checkHelm() {
   if [ $isExist ]; then
     true # Placeholder, do nothing
   else
-    echo "ERROR: Helm command not found, Please install helm v3."
+    echo "[ERROR] Helm check failed: Helm command not found, Please install helm v3."
     exit
   fi
   # check version
   # version will be like:
   # Version:"v3.15.2"
   regex="Version:\"(v[0-9]\.[0-9]+\.[0-9])\""
   version=$(helm version)
```

Comment:
I think we needs check if exist `helm` first.

Reply:
This section checks if Helm exists:

```bash
isExist=$(command -v helm)
if [ "$isExist" ]; then
```

I've unified the usage of `which` and `command -v` to use `command -v` for consistent style.

```bash
if command -v helm >/dev/null 2>&1; then
```